src/meta/src/stream/stream_graph/fragment.rs

```diff
@@ -194,6 +194,28 @@ impl BuildingFragment {
 
         table_columns
     }
+
+    pub fn has_arrangement_backfill(&self) -> bool {
```

Comment:
Can we use `visit_stream_node` here?

Reply:
Not really, `visit_stream_node`'s interface is as such:
```
pub fn visit_stream_node<F>(stream_node: &mut StreamNode, mut f: F)
```

It mutates the stream node itself.

What we want to do here is to traverse the stream node and extract some value.

We can introduce a separate interface like:
```
pub fn stream_node_has(stream_node: &mut StreamNode, f: fn(...) -> bool) -> bool
```

If we want something more general in the future.